runtime/rastrace/trcengine.c

```diff
@@ -493,7 +493,7 @@ J9VMDllMain(J9JavaVM *vm, IDATA stage, void *reserved)
 			}
 		}
 
-		/* Normal exit - its not safe to free this stuff if we are ab'ending.
+		/* Normal exit - its not safe to free this stuff if we are abending.
```

Comment:
`abend` while not a word I particularly like is a word-ish thing that I encounter elsewhere.